src/frontend/e2e/tests/sample-page.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { expect, test } from "@playwright/test";
+import { getByTestID, getByText } from "../utils/selectors";
+import path from "path";
+import dotenv from "dotenv";
+import fs from "fs";
+
+dotenv.config({ path: path.resolve(`.env.${process.env.NODE_ENV}`) });
+
+const tableHeaders = [
+  "Private ID",
+  "Public ID",
+  "Collection Date",
+  "Lineage",
+  "Upload Date",
+  "Collection Location",
+  "Sequencing Date",
+  "GISAID",
+];
+const api = `${process.env.BASEAPI}/v2/orgs/${process.env.GROUPID}/pathogens/SC2/samples/`;
+const tAndCSelector =
```

Comment:
what is tandc? maybe something more readable would be better

Reply:
Fixed. This accept site cookies button